fog/report/resolver/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "mc-fog-report-resolver"
+version = "2.0.0"
+authors = ["MobileCoin"]
+edition = "2021"
+
+[features]
+default = []
+automock = ["mockall"]
+
+[dependencies]
+mc-account-keys = { path = "../../../account-keys" }
+mc-attest-verifier = { path = "../../../attest/verifier", default-features = false }
+mc-fog-report-types = { path = "../types" }
+mc-fog-report-validation = { path = "../validation" }
+mc-fog-ingest-report = { path = "../../ingest/report" }
```

```suggestion
mc-fog-ingest-report = { path = "../../ingest/report" }
mc-fog-report-types = { path = "../types" }
mc-fog-report-validation = { path = "../validation" }
```

I think the linter just wants these packages to be sorted alphabetically